.github/workflows/windows_release.yml

```diff
@@ -33,6 +33,10 @@ jobs:
     - name: Add MikTeX binary folder to path
       run: echo "::add-path::/C/Program Files/MikTeX/miktex/bin/x64"
 
+    - name: Build Test Document (will install missing packages)
+      working-directory: ${{runner.workspace}}/EnergyPlus/doc/test
+      run: xelatex dependencies.tex
```

Comment:
We only install the basic MikTeX setup.  Because of this, some of the packages we need are missing, but get installed on the fly.  During our documentation builds, occasionally the build would fail because what appeared to be a miktex conflict where multiple threads were trying to build the docs and update packages at the same time.  What I did here was just create a minimal TeX file that simply includes the required LaTeX packages.  We can then build this once ahead of time, which will install the dependencies in a single thread, then we are good to build EnergyPlus itself in a multithreaded environment.